src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -356,15 +356,25 @@ short Terminal::GetBufferHeight() const noexcept
 }
 
 // _ViewStartIndex is also the length of the scrollback
-int Terminal::_ViewStartIndex() const noexcept
+int Terminal::ViewStartIndex() const noexcept

```

Comment:
It strikes me as odd that the ViewStartIndex has no _, implying public. But the matching ViewEndIndex is still prefixed with _, implying private. Do you not need both?

Reply:
changed _ViewEndIndex to public and removed the "_"